examples/packages/single-language/python-project/pyproject.toml

```diff
@@ -6,6 +6,13 @@ build-backend = "setuptools.build_meta"
 name = "my-tool"
 description = "my tool"
 version = "1.0.0"
+
+requires-python = ">=3.11.0"
 dependencies = [
-  "requests"
+	"requests",
 ]
+
+[tool.dream2nix]
+project-folder = "my_project"
+python-version = "python311"
+pypi-snapshot-date = "2024-03-01"
```

Comment:
While I like this idea in general, I would like to implement this in a more generic way, so that everything from, for example `tool.draem2nix.config` gets mapped to the package's config.

For now I'd prefer not to add these, because we will have to deprecate them again once the generic implementation is in place, and users will face breakages.